### DISCUSSION TITLE
Apple Devices

### DISCUSSION THREAD

**Discussion by giscus** [AwaitQuality/windowsonarm#25] [General]
# Apple Devices

Check if Apple Devices is ARM ready on Windows and can be used on Windows ARM devices such as the Surface with the Snapdragon X Elite.

https://windowsonarm.org/5598b6c9-ae3c-4b43-aa15-f401811b6697

**Comment by talynone** [comment]
Some apps that are blocked are just artificially blocked and work fine, the following method may work:

(use the following url to feed into downloader: https://apps.microsoft.com/detail/9np83lwlpz9k?hl=en-US&gl=US)

https://old.reddit.com/r/Surface/comments/1djltm6/the_new_windows_apple_music_app_works_on_windows/
